### PR TITLE
Changing documentation structure

### PR DESCRIPTION
**Pull request by paulfantom**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

PR implements what was discussed in #1005

## Type of change

_What type of changes does your code introduce to the kube-prometheus? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. Later this will be copied to the changelog file._

```release-note

```

### CODE REVIEW

File: docs/update.md
Comment (ArthurSens) [inline docs/update.md:10]:
```suggestion
$ go install -u github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
```

File: docs/update.md
Comment (ArthurSens) [inline docs/update.md:23]: Could "compiling" and "apply the kube-prometheus stack" have a link to the appropriate place?

**Review by ArthurSens — APPROVED** [review]
The README is looking super clean now, thanks a lot for tackling this! 🧹 

I've just noticed some super small nits :)

File: docs/update.md
Comment (paulfantom) [reply docs/update.md]: Fixed.

**Comment by paulfantom** [comment]
@ArthurSens I included fixes to the issues mentioned during the review. Feel free to merge this if you think it is ready.

**Comment by metalmatze** [comment]
Really nice! :+1: 
Make sure the website doesn't break too much with these changes. Maybe we need to add some new pages to the navigation there? Not sure myself right now.

**Comment by paulfantom** [comment]
Good call with website! I will take a look at that after we merge this PR.

**Comment by ArthurSens** [comment]
Thanks again :)

**Pull request merged**
